package.json

```diff
@@ -82,19 +78,15 @@
     "fuse-concat-with-sourcemaps": "^1.0.5",
     "glob": "^7.1.1",
     "ieee754": "^1.1.8",
-    "inquirer": "^3.0.6",
```

Comment:
Are you sure it's not in use anywhere?

Reply:
Yep, i have search in all files on github and in the project

Reply:
👍